### PR TITLE
Add gheb\docusign-bundle

### PR DESCRIPTION
**Pull request by GregoireHebert**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: gheb/docusign-bundle/4.5/config/packages/docusign.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+docusign:
+    access_token: "YourAccessToken"
```
Comment (vincentchalamon) [inline gheb/docusign-bundle/4.5/config/packages/docusign.yaml]: Please add a comment with the link where to generate the token: https://developers.docusign.com/oauth-token-generator

**Comment by GregoireHebert** [comment]
This PR is still WIP waiting for a new recipe configuration.

File: gheb/docusign-bundle/5.0/config/packages/docusign.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    mode: embedded # Mode used to sign (remote or embedded)
+
+    # Authentication credentials to log into docusign.
+    auth_jwt:
+        private_key: "%env(DOCUSIGN_PRIVATE_KEY_PATH)%"
+        integration_id: "%env(DOCUSIGN_INTEGRATION_ID)%"
+        user_guid: "%env(DOCUSIGN_USER_GUID)%"
+
+    # Your DocuSign account id
+    account_id: "%env(DOCUSIGN_ACCOUNT_ID)%"
```
Comment (vincentchalamon) [inline gheb/docusign-bundle/5.0/config/packages/docusign.yaml]:
```suggestion
    account_id: "%env(int:DOCUSIGN_ACCOUNT_ID)%"
```

File: gheb/docusign-bundle/5.0/config/packages/docusign.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+docusign:
+    demo: "%kernel.debug%"
+    mode: embedded # Mode used to sign (remote or embedded)
+
+    # Authentication credentials to log into docusign.
+    auth_jwt:
+        private_key: "%kernel.project_dir%/%CONFIG_DIR%/jwt/docusign.pem"
+        integration_key: "51b176de-3298-460b-8201-f73e323eee14" # Replace by yours
+        user_guid: "51b176de-3298-460b-8201-f73e323eee14" # Replace by yours
```
Comment (Nyholm) [inline gheb/docusign-bundle/5.0/config/packages/docusign.yaml]: These should use environment variables (as comments)

File: gheb/docusign-bundle/5.0/config/packages/docusign.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+    # Your DocuSign account id
+    account_id: 1234567
```
Comment (Nyholm) [inline gheb/docusign-bundle/5.0/config/packages/docusign.yaml]: ??

File: gheb/docusign-bundle/5.0/config/packages/docusign.yaml (hunk 3 above)
Comment (GregoireHebert) [reply gheb/docusign-bundle/5.0/config/packages/docusign.yaml]: Well to be honest at first they were set as such but using validation within the configuration prevented their usage. But in the end, it make more sense for this kind of data to be environment variables (secrets) so I backed up a little on the code by removing the checks.
So yes, I can change that back.

Thanks for the review @Nyholm !

**Pull request merged**